src/fixtures-loader.coffee

```diff
@@ -37,19 +34,14 @@ module.exports =
     _.forEach models, (model) =>
       purgeModelPromises.push @purgeModel(model)
 
-    q.all purgeModelPromises
+    Promise.all purgeModelPromises
 
 
   purgeModel: (model) ->
-    purgingModel = q.defer()
-
-    model.destroyAll (err) ->
-      if err
-        purgingModel.reject err
-      else
-        purgingModel.resolve()
-
-      purgingModel.promise
```

Comment:
Be careful. Since the return of the promise is in the callback, I've found errors to be silently thrown in the void.